controls/SV-221587.rb

```diff
@@ -34,4 +34,9 @@
   tag legacy: ["SV-94633","V-79929"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
-end
\ No newline at end of file
+  
+  describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome') do
+    it { should have_property '\"PromptForDownloadLocation\"'}
```

Comment:
The combination of backslash followed by double quotes,`\"`, is not necessary. It checks for `\"PromptForDownloadLocation\"` as opposed to `PromptForDownloadLocation` as the property. Similarly, it checks for `\"1\"` as opposed to `1` as the value. Consider removing this combination `\"` to fix the syntax.